src/Controls/src/Core/Element/Element.cs

```diff
@@ -333,30 +333,39 @@ internal Element ParentOverride
 		}
 
 		WeakReference<Element> _realParent;
-		/// <summary>For internal use by .NET MAUI.</summary>
-		[EditorBrowsable(EditorBrowsableState.Never)]
-		public Element RealParent
+		Element GetRealParent(bool logWarningIfParentHasBeenCollected = true)
```

Comment:
Because can return null, maybe a better name for the method would be `TryGetRealParent`